css/properties/paint-order.json

```diff
@@ -12,7 +12,7 @@
               "version_added": false
             },
             "edge": {
-              "version_added": false
+              "version_added": "17"
```

Comment:
I can't find a proof or any additional hints for this. https://developer.microsoft.com/en-us/microsoft-edge/platform/usage/css/paint-order/ only mentions Chrome and Firefox, but maybe you're correct and it was added recently in version 17. :man_shrugging:

Reply:
I tested https://developer.mozilla.org/en-US/docs/Web/CSS/paint-order#Result in Edge 16-18 and they all look the same, as if `paint-order` isn't supported.

However, turns out the example is wrong. This property only works for SVG. Updated example:
https://codepen.io/foolip/pen/NJZPWg?&editable=true

With this example I can confirm that i didn't work in Edge 16 and started working in 17.

I've made an edit to the MDN article itself to use this updated example and requested technical review.

Reply:
@Elchi3 I see https://developer.mozilla.org/en-US/docs/Web/CSS/paint-order#Result is updated now, but the example is broken. Don't SVG examples work, or did I mess up the markup?

Reply:
I've fixed this. There is apparently no SVG code highlighting that works with this. Other SVG samples use "brush:html". :man_shrugging: